src/utils/images/controls/play.svg

```diff
@@ -1,3 +1,38 @@
-<svg width="45" height="46" viewBox="0 0 45 46" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M22.3564 0.591568C10.0556 0.591568 0.0722656 10.6337 0.0722656 23.007C0.0722656 35.3802 10.0556 45.4223 22.3564 45.4223C34.6572 45.4223 44.6405 35.3802 44.6405 23.007C44.6405 10.6337 34.6572 0.591568 22.3564 0.591568ZM17.8996 33.0939L17.8996 12.92L31.2701 23.007L17.8996 33.0939Z" fill="white"/>
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Yeah i was using the svgs provided in figma as well. But now we need to change the background colors of the buttons in the `active` state. To achieve that, we need to change the color of the div wrapping these svgs. I couldn't achieve that with this particular one because of how the play button is designed. We need an 'inverted' version of it. Ideally, the client should provide us with one but this looks pretty good too. 

Client gave us this:
![image](https://user-images.githubusercontent.com/26784326/116435382-b7df6900-a864-11eb-8e7d-a547ce9f64a5.png)

We need this:
![image](https://user-images.githubusercontent.com/26784326/116435415-bf9f0d80-a864-11eb-99fa-eaa991983c6d.png)

Reply:
I am assuming the circles are part of the buttons and they become faded as well once clicked. Let me confirm this.